bounties/level-1/create-bounties.md

```diff
@@ -0,0 +1,33 @@
+# Creating Bounty Suggestions
+
+| Hunter | Agent
+| :- | :-
+| 500 XLM per bounty | 0 Credits (success will be determined by PR acceptance)
```

Comment:
Let's decrease this to 200 XLM. Bounties haven't been as tricky to create as I first suspected. I'll honor the 500 XLM retrospectively but ongoing this should be lower.